docs/application/native/guides/machine-learning/machine-learning-inference.md

```diff
@@ -482,6 +482,11 @@ For more information about the pipeline states, see [GStreamer guide](https://gs
     ml_tensors_info_destroy (info);
     ```
 
+    If you want to use tensors which dimension is higher than 4, use the following function.
```

Comment:
```suggestion
    If you want to use tensors whose dimension is higher than 4, use the following function:
```

Reply:
Thank you for the suggestion :)
Updated it